irohad/torii/impl/command_service.cpp

```diff
@@ -130,4 +136,117 @@ namespace torii {
     Status(*request, *response);
     return grpc::Status::OK;
   }
+
+  void CommandService::StatusStream(
+      iroha::protocol::TxStatusRequest const &request,
+      grpc::ServerWriter<iroha::protocol::ToriiResponse> &response_writer) {
+    auto resp = cache_->findItem(request.tx_hash());
+    if (resp) {
+      if (resp->tx_status()
+              == iroha::protocol::TxStatus::STATELESS_VALIDATION_FAILED
+          or resp->tx_status()
```

Comment:
Also, I guess more clear to hold all values into collection and perform `OR` function (accumulate for example) for the collection